test/trento_web/channels/activity_log_channel_test.exs

```diff
@@ -0,0 +1,26 @@
+defmodule TrentoWeb.ActivityLogChannelTest do
+  use TrentoWeb.ChannelCase
+
+  test "Socket users can only join the activity log channel" do
```

Comment:
Possible, yes. But if we add this assert, we would be slowing down the test suite by 60s. I am not convinced this addition would be worth the wait. Unless the refresh interval gets parameterized in some form.

Reply:
> we would be slowing down the test suite by 60s
>
I hear you, and parametrization of the interval sounds like might help.

Rather than waiting I wan wondering about a way to assert that `Process.send_after(self(), :after_join, 60_000)` has been called, hence we have something in queue for the process, but not wait for it.

Up to you. Whatever works best, but I believe that since that interval is part of the _core_ of the functionality I'd say it is valuable being able to test it.

Reply:
On second thought, I don't think this is something that should be tested at this level ( the fact that `Process.send_after` does what it should do after the given time interval or that it queues something in the process mail box should not be subject to a test in application code; in any case I don't think it possible to make such asserts without doing some VM calls tracing). I believe the overall behaviour (that the users list is auto-updated after a certain interval) is better tested at the integration layer (the end to end tests, or similar, when that is looked into).

Reply:
I see. What about parametrization of the delay?

Reply:
We could pull the delay as environmental parameter, pulled from the Application env. Concretely, default delay will remain 60s but in tests it will be 1ms. How does this sound?

Reply:
Sounds reasonable. Do not bother making that configurable by environment variable. As long as we have a default and we can override for testing purposes, I guess it should be enough.